crates/dscvr-telemetry-util/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+pub mod axum {
+    use axum::{extract::MatchedPath, middleware::Next, response::Response, routing::get, Router};
+    use http::Request;
+    use metrics_exporter_prometheus::{BuildError, PrometheusBuilder};
+    use std::time::Instant;
+
+    // Takes an existing axum router, installs the prometheus metrics recorder and
+    // injects the metrics endpoint into the router after the handler layer is installed so that
+    // `/metrics` route itself is not included in the routing layer metrics measured
+    pub fn install_axum_metrics(app: Router) -> Result<Router, BuildError> {
```

Comment:
should we just call this `install_metrics` since it's scoped to `axum` module now? Either that or please make `track_metrics` consistent with this